frontend/pages/admin/IntegrationsPage/cards/Mdm/Mdm.tsx

```diff
@@ -0,0 +1,292 @@
+import React, { useContext, useState } from "react";
+import { useQuery } from "react-query";
+import FileSaver from "file-saver";
+
+import { AppContext } from "context/app";
+import { NotificationContext } from "context/notification";
+
+import mdmAppleAPI from "services/entities/mdm_apple";
+import mdmAppleBmAPI from "services/entities/mdm_apple_bm";
+import { IMdmApple, IMdmAppleBm } from "interfaces/mdm";
+
+import Button from "components/buttons/Button";
+import CustomLink from "components/CustomLink";
+import Spinner from "components/Spinner";
+import DataError from "components/DataError";
+import RequestModal from "./components/RequestModal";
+
+// MDM TODO: key validation?
+// import { isValidKeys } from "../../..";
+
+const baseClass = "mdm-integrations";
+
+const readableDate = (date: string) => {
+  const dateString = new Date(date);
+
+  return new Intl.DateTimeFormat("en-US", {
```

Comment:
A really simple improvement, but we can use `navigoator.language` instead of `
to get the preferred language of the user

Reply:
Oh cool! I did a lot of googling to figure out a way to make the date appear the way the Figma wanted. Thanks for adding to my understanding!